server/honeybadger/plugins.py

```diff
@@ -10,8 +11,9 @@ def get_coords_from_google(aps):
         data['wifiAccessPoints'].append(ap.serialized_for_google)
     data_json = json.dumps(data)
     headers = {'Content-Type': 'application/json'}
-    request = urllib2.Request(url, data=data_json, headers=headers)
-    response = urllib2.urlopen(request)
+    request = urllib.request.Request(url, data="{}".encode(), headers=headers)
```

Comment:
Why do you need to encode this? A unicode object should be fine here shouldn't it?

Reply:
When not calling encode, I was getting type errors. Python 3's urllib module expected a bytes-like object. Calling encode converts it from a string object to a bytes object.

Reply:
That explains all of the encoding and decoding then. What a pain. See the below comment about using the requests module. We won't have this issue with it.

Reply:
The replacement of urllib with requests shouldn't be terribly painful, and that can be the next task I work on. I had not used urllib before, and I definitely prefer the requests library.